src/Tribe/Capabilities.php

```diff
@@ -15,24 +47,64 @@ class Tribe__Events__Capabilities {
 			'delete_private_posts',
 			'delete_published_posts',
 			'publish_posts',
-		),
-		'author' => array( // full permissions for content the user created
+		],
+		// Full permissions for content the user created.
+		'author'      => [
 			'read',
 			'edit_posts',
 			'edit_published_posts',
 			'delete_posts',
 			'delete_published_posts',
 			'publish_posts',
-		),
-		'contributor' => array( // create, but not publish
+		],
+		// Create, but not publish.
+		'contributor' => [
 			'read',
 			'edit_posts',
 			'delete_posts',
-		),
-		'subscriber' => array( // read only
+		],
+		// Read only.
+		'subscriber'  => [
 			'read',
-		),
-	);
+		],
+	];
+
+	/**
+	 * @deprecated TBD
+	 *
+	 * @var bool
+	 */
+	public $set_initial_caps = false;
+
+	/**
+	 * Initialize Setting the Capabilities.
+	 *
+	 * @since TBD
+	 */
+	public function init_set_caps() {
+
+		set_transient( $this->key_delayed_capabilities_update, 'yes', 0 );
```

Comment:
Transients should have an expiration. One day seems a reasonable one.

Reply:
There is no need to have an expiration, since it will be deleted on the next load of the page.